nautobot/docs/additional-features/jobs.md

```diff
@@ -553,57 +553,113 @@ Provision of user input (`data` values) via the CLI is not supported at this tim
-
-Because of the way `run_job()` works, which is somewhat complex behind the scenes, there are a few things you'll need to do in your test code:
-
-1. Inherit from `django.test.TransactionTestCase` instead of `django.test.TestCase`. (Refer to the [Django documentation](https://docs.djangoproject.com/en/stable/topics/testing/tools/#provided-test-case-classes) if you're interested in the differences between these classes.)
-2. Set your test class's `databases` attribute to `("default", "job_logs")`. (`job_logs` is a proxy connection to the same (`default`) database that's used exclusively for Job logging.)
-
-!!! tip
-    Additionally, *if your test case needs to be backwards-compatible with test execution against Nautobot 1.2.x*, you should add the following block to your test module:
-
-        from django.conf import settings
+While individual methods within your Job can and should be tested in isolation, you'll likely also want to test the entire execution of the Job. Nautobot 1.3 introduced a couple of changes to the way Jobs are handled. As a consequence, the recommended job testing approach also differs between 1.2.x and 1.3.x and later.
+
+### 1.3.x and later
+
+The simplest way to test the entire execution of Jobs from 1.3 on is via calling the `nautobot.utilities.testing.run_job_for_testing()` method, which is a helper wrapper around the `run_job` function used to execute a Job via Nautobot's Celery worker process.
+
+Because of the way `run_job_for_testing` and more specifically `run_job()` works, which is somewhat complex behind the scenes, you need to inherit from `nautobot.utilities.testing.TransactionTestCase` instead of `django.test.TestCase` (Refer to the [Django documentation](https://docs.djangoproject.com/en/stable/topics/testing/tools/#provided-test-case-classes) if you're interested in the differences between these classes - `TransactionTestCase` from Nautobot is a small wrapper around Django's `TransactionTestCase`).
+
+### 1.2.x
+
+If your test case needs to be backwards-compatible with test execution against Nautobot 1.2.x, you need to handle a couple more things manually:
+
+- Set up the `"job_logs"` database correctly for testability
+  ```
+  from django.conf import settings
+
+  if "job_logs" in settings.DATABASES:
+      settings.DATABASES["job_logs"] = settings.DATABASES["job_logs"].copy()
+      settings.DATABASES["job_logs"]["TEST"] = {"MIRROR": "default"}
+  ```
+- Replicate the behavior of `run_job_for_testing` manually so that your test execution most closely resembles the way the celery worker would run the test and set up the `databases` field on the test class correctly
```

Comment:
```suggestion
- Replicate the behavior of `run_job_for_testing` manually so that your test execution most closely resembles the way the celery worker would run the test:
```